src/zif_abapgit_environment.intf.abap

```diff
@@ -34,4 +34,10 @@ INTERFACE zif_abapgit_environment
   METHODS is_variant_maintenance
     RETURNING
       VALUE(rv_is_variant_maintenance) TYPE abap_bool.
+  METHODS init_parallel_processing
+    IMPORTING
+      iv_group                      TYPE rzlli_apcl
```

Comment:
so the INTF is ABAP cloud compatible, 

```suggestion
      iv_group                      TYPE clike
```

Reply:
Thx. Fixed.
How can I detect these kind of problems in advance?